internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -92,6 +94,17 @@ type Opts struct {
 	store               store.AtlasClusterQuickStarter
 }
 
+type DefaultValues struct {
```

Comment:
```suggestion
type Quickstart[Options|Values] struct {
```

Reply:
Linter won't let me call it `quickstart.Quickstart` ):

Reply:
would it still complain if you make it private (which we should `quickstart`? if still complains then `formValues` or `formOptions` or just `form` works as well

Reply:
thanks :) it's private now